assets/src/helpers.js

```diff
@@ -18,6 +18,8 @@ import { getColorClassName, getColorObjectByAttributeValues, RichText } from '@w
 /**
  * Internal dependencies
  */
+// eslint-disable-next-line no-unused-vars
+import store from './stores/amp-story';
```

Comment:
Does `import './stores/amp-story';` alone just work to remove the need for `eslint-disable`?

Reply:
Ah, good point, it should work 💡